pom.xml

```diff
@@ -120,7 +120,7 @@
     <sbt.project.name>spark</sbt.project.name>
     <slf4j.version>1.7.30</slf4j.version>
     <log4j.version>1.2.17</log4j.version>
-    <hadoop.version>3.3.1</hadoop.version>
+    <hadoop.version>3.3.2</hadoop.version>
```

Comment:
should we update https://github.com/apache/spark/pull/34830#discussion_r767148540 together?

Reply:
Good point. Will do.

Reply:
> should we update [#34830 (comment)](https://github.com/apache/spark/pull/34830#discussion_r767148540) together?

+1 on. this